src/DynamoUtilities/AssemblyHelper.cs

```diff
@@ -1,12 +1,24 @@
 ﻿using System;
```

Comment:
`AssemblyHelper` is used for test cases. Each test fixture `Setup` now creates an instance of this class, registering to `AppDomain.CurrentDomain.AssemblyResolve` event at creation time, and unregistering from this event once `TearDown` happens.